modules/table/src/main/java/org/apache/ignite/internal/table/distributed/SortedIndexLocker.java

```diff
@@ -83,26 +68,7 @@ public SortedIndexLocker(UUID indexId, int partId, LockManager lockManager, Sort
         this.storage = storage;
         this.indexRowResolver = indexRowResolver;
 
-        this.positiveInf = createInfiniteBoundary(partId, indexId);
-    }
-
-    /**
-     * Creates a tuple for positive infinity boundary.
-     *
-     * @param partId Partition id.
-     * @param indexId Index id.
-     * @return Infinity binary tuple.
-     */
-    private static BinaryTuple createInfiniteBoundary(int partId, UUID indexId) {
-        var binarySchema = BinaryTupleSchema.createSchema(INFINITY_TUPLE_SCHEMA, new int[]{
-                INFINITY_TUPLE_SCHEMA.column("indexId").schemaIndex(),
-                INFINITY_TUPLE_SCHEMA.column("partId").schemaIndex()
-        });
-
-        return new BinaryTuple(
-                binarySchema,
-                new BinaryTupleBuilder(binarySchema.elementCount(), false).appendUuid(indexId).appendInt(partId).build()
-        );
+        this.positiveInf = new Integer(partId);
```

Comment:
@Deprecated(since="9", forRemoval = true)
    public Integer(int value) {

Reply:
Replace to valueOf()